kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -57,23 +65,37 @@
       return {
         user: {
           username: '',
-          password: '',
           full_name: '',
+          password: '',
+          confirm_password: '',
         },
         role: 'learner',
       };
     },
+    computed: {
+      checkPassword() {
+        // if (this.user.password.length =< 9) {
+        //   return 'Error: too short';
+        // }
+        if (this.user.confirm_password && this.user.password !== this.user.confirm_password) {
+          return 'passwords not matched';
+        }
+        return '';
+      },
+    },
     methods: {
       createNewUser() {
-        this.user.facility = this.facility;
-        // using promise to ensure that the user is created before closing
-        // can use this promise to have flash an error in the modal?
-        this.createUser(this.user, this.role).then(() => {
-          for (const userProp of Object.getOwnPropertyNames(this.user)) {
-            this.user[userProp] = '';
-          }
-          this.$refs.modal.closeModal();
-        });
+        if (this.user.password && this.user.confirm_password === this.user.password) {
```

Comment:
I feel like this could be streamlined significantly. I might take a look at refactoring this after MVP